src/server/object_services/schemas/nsfs_bucket_schema.js

```diff
@@ -42,7 +42,13 @@ module.exports = {
             $ref: 'common_api#/definitions/fs_backend'
         },
         s3_policy: {
-            $ref: 'common_api#/definitions/bucket_policy',
+            oneOf: [{
+                $ref: 'common_api#/definitions/bucket_policy',
+            }, {
+                type: 'string',
+                enum: ['']
```

Comment:
here again please don't use this hack and delete the bucket policy as I did in the PR attached in the previous comment

Reply:
I'm not sure what you're suggesting.
In case someone created a bucket with a bucket policy and tries to delete it - what do you expect the bucket policy to be? (with current code it is an empty string).

Reply:
undefined